zookeeper-command-line-client/src/main/sh/vespa-zkflw

```diff
@@ -77,6 +77,24 @@ export ROOT
 
 # END environment bootstrap section
 
+# Defaults
+server="localhost"
+port="2181"
+command="stat"
+
+if [ "$#" -eq 1 ]; then
+    command=$1
+elif [ "$#" -eq 2 ] || [ "$#" -gt 3 ]; then
+    echo "Need 0, 1 (command) or 3 arguments (host, port, command)"
+    exit 1
+elif [ "$#" -eq 3 ]; then
+    server=$1
+    port=$2
+    command=$3
+fi
+
+args=("$server" "$port" "$command")
```

Comment:
Remove this line, as `args` is set above.

Reply:
We always need to use all 3 arguments, so cannot do it this way when there is just 0 or 1 arguments

Reply:
:+1: Got it